wipp-backend-data/src/main/java/gov/nist/itl/ssd/wipp/backend/data/imagescollection/ImagesCollectionDataHandler.java

```diff
@@ -93,6 +93,13 @@ public String exportDataAsParam(String value) {
         }
         // else return the path of the regular images collection
         else {
+            if (imagesCollectionRepository.findById(imagesCollectionId).isPresent()) {
```

Comment:
imagesCollectionRepository.findById(imagesCollectionId) could be called only once before the if and stored in a variable (to avoid calling it twice)